docker/UCXPy-MOFED.dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG CUDA_VERSION=11.5.2
+ARG DISTRIBUTION_VERSION=ubuntu20.04
+FROM nvidia/cuda:${CUDA_VERSION}-devel-${DISTRIBUTION_VERSION}
+
+# Make available to later build stages
+ARG DISTRIBUTION_VERSION
+# Should match host OS OFED version (as reported by ofed_info -s)
+ARG MOFED_VERSION=5.3-1.0.5.0
+# Tag to checkout from UCX repository
+ARG UCX_VERSION_TAG=v1.12.1
+# Where to install conda, and what to name the created environment
+ARG CONDA_HOME=/opt/conda
+ARG CONDA_ENV=ucx
+# Name of conda spec file in the current working directory that
+# will be used to build the conda environment.
+ARG CONDA_ENV_SPEC=ucx-py-cuda11.5.yml
+
+ENV CONDA_ENV="${CONDA_ENV}"
+ENV CONDA_HOME="${CONDA_HOME}"
+
+# Where cuda is installed
+ENV CUDA_HOME="/usr/local/cuda"
+
+ADD https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh /miniconda.sh
```

Comment:
`ADD` does a lot of magic and is generally discouraged these days. It would probably be preferable to curl this instead as part of the same `RUN` that calls and removes it.